application/src/main/java/org/opentripplanner/standalone/config/routerconfig/updaters/MqttGtfsRealtimeUpdaterConfig.java

```diff
@@ -21,12 +23,43 @@ public static MqttGtfsRealtimeUpdaterParameters create(String configRef, NodeAda
         .since(V2_0)
         .summary("Whether to match trips fuzzily.")
         .asBoolean(false),
+      c
+        .of("forwardsDelayPropagationType")
+        .since(V2_8)
+        .summary("How forwards propagation should be handled.")
+        .description(
+          """
```

Comment:
If you wish, you can make `ForwardDelayPropagationType` a `DocumentedEnum` and automate the generation of this documentation, like this:

https://github.com/opentripplanner/OpenTripPlanner/blob/d11efcf26fc839cbe5a767a14824909529b08b19/application/src/main/java/org/opentripplanner/standalone/config/routerconfig/ServerConfig.java#L51-L53